### ISSUE TITLE
RFI: How to edit the react-admin installation inside the containers

### ISSUE THREAD

**Issue by caston1981** [francoisruty/fruty_react-admin#3]
Hi Fruty,

Really sorry to bother you again. Great project and thank you for making it. I have it all up an running on a virtualbox on a second Ubuntu machine so I don't get it mixed up (clean environment) with other projects I have been testing. 

I am just not sure how I continue to develop the react-admin installation. In the docker-compose.yml it has a line: REACT_EDITOR: atom

I have both visual studio code and atom installed on my main machine. Do I use features within one of these program  to edit the code in the docker containers? If so am I editing both the front and back?

I have been following at the tutorial level (https://marmelab.com/react-admin/Tutorial.html) but then I'm not sure how to apply and build upon this knowledge to work with the dockerised and postgres connected version here.

I don't expect you to tutor me or be there for my learning curve I am just asking for a point in the right direction. Perhaps you can link me to literature and resources you used to get to this point. I will learn what is required in my own time and won't keep bothering you with questions.

Thank you very much,

Chris

**Comment by francoisruty** [comment]
Hello, you don't need ATOM, you can ignore this env variable

Currently the docker-compose has volume mappings which map the source code directly inside the docker containers.
So you can edit the source code files directly with your IDE, and hot reloading shall happen on back+front and update what you see in your browser.
This is a dev environment, not a production environment, so you're all good to go

Front code is in "front" folder (this is where there is react-admin stuff), backend code is in "back" folder

When you follow react-admin tutorials, just edit the code in "front" folder

**Comment by caston1981** [comment]
Hi Fruty,

Thank you very much for that. I have forked it (https://github.com/affluent-bilby-classifieds/fruty_ra-cb-menu) as it's going to have an editable menu (currently a json file) for a high-school friends' (twins actually) business called Compton burgers.

I also noticed the back-end mentions Express.js which I haven't even looked at yet. I guess I will learn more as I dig deeper. Thanks again for your help.
